packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -26,6 +26,7 @@ describe('network stubbing', { retries: 15 }, function () {
 
   beforeEach(function () {
     cy.spy(Cypress.utils, 'warning')
+    cy.visit('/fixtures/empty.html')
```

Comment:
We'll want to remove this. This should have only been necessary in Electron 28 - that's the assumption anyway. See  https://github.com/cypress-io/cypress/pull/28959/files#r1603224984